docs/releases/v1.15.0.md

```diff
@@ -0,0 +1,64 @@
+# Networking Core Module Release 1.15.0
+
+Welcome to the latest release of the `Networking` module of [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution) maintained by team SIGHUP.
+
+This minor release updates some components and adds support to Kubernetes 1.27.
+
+## Component Images 🚢
+
+| Component         | Supported Version                                                                | Previous Version |
+| ----------------- | -------------------------------------------------------------------------------- | ---------------- |
+| `calico`          | [`v3.26.1`](https://projectcalico.docs.tigera.io/archive/v3.26/release-notes/)   | `v3.25.0`        |
+| `cilium`          | [`v1.14.3`](https://github.com/cilium/cilium/releases/tag/v1.14.3)               | `v1.13.1`        |
+| `ip-masq`         | [`v2.8.0`](https://github.com/kubernetes-sigs/ip-masq-agent/releases/tag/v2.5.0) | No update        |
+| `tigera-operator` | [`v1.30.4`](https://github.com/tigera/operator/releases/tag/v1.30.4)             | `v1.29.0`        |
+
+> Please refer the individual release notes to get detailed information on each release.
+
+## Update Guide 🦮
+
+### Process
+
+If you are using Cilium, read the steps [below](#cilium-upgrade) before proceeding.
+
+1. Just deploy as usual:
+
+```bash
+kustomize build katalog/calico | kubectl apply -f -
+# OR
+kustomize build katalog/tigera/on-prem | kubectl apply -f -
+# OR
+kustomize build katalog/cilium | kubectl apply -f -
+```
+
+#### Cilium upgrade
+Cilium suggested path expect a pre-flight check to be run before any upgrade.
+
+1. Create the resources for the check
+```bash
+kubectl create -f katalog/cilium/tasks/preflight.yaml
+```
+
+2. Ensure that the number of READY pods is the same number of Cilium pods running.
```

Comment:
"Make sure that the number of READY pods is the same as the number of RUNNING Cilium pods."

Reply:
```suggestion
2. Make sure that the number of READY pods is the same as the number of RUNNING Cilium pods.
```